lib/pbench/test/unit/server/test_schema.py

```diff
@@ -58,7 +59,7 @@ def test_failed_conversions(self, test, monkeypatch):
         def not_ok(user: str) -> str:
             raise UnknownUser()
 
-        monkeypatch.setattr(Auth, "validate_user", not_ok)
+        monkeypatch.setattr(Auth, "verify_user", not_ok)
```

Comment:
As I pointed out in my previous comment, the signature for `not_ok()` is incorrect -- it should match `verify_user()`.